### DISCUSSION TITLE
Superset SDK hide options in chart controls

### DISCUSSION THREAD

**Discussion by jiankailee** [apache/superset#24402] [Q&A / Help]
Hi all,

I am successfully integrated the embedded superset in my angular UI with guest token approach. In the dashboard UI config, it allows me to set to true or false in hideChartControls. However, I would like to hide `Edit Chart`, `View query`, `View as table` in Superset embedded SDK. Is that possible? What settings do I need to hide these options?

Thank you.
![image](https://github.com/apache/superset/assets/35210913/218eea9f-f20a-4fe7-a54a-8fd94b695933)

**Comment by mikkhait** [comment]
hey @jiankailee very good question! Were you able to find an answer?

In my case, I'm trying to give a user of an embedded pie chart an option to see the data in a table
## Non-embedded
![CleanShot 2024-01-18 at 12 26 18@2x](https://github.com/apache/superset/assets/1232731/69e9c6d6-6af0-4bb8-89b1-c575409c48e1)

## Embedded
![CleanShot 2024-01-18 at 12 26 56@2x](https://github.com/apache/superset/assets/1232731/6cc0de26-d28a-4cb8-8c47-6772a8fc55c2)

**Reply by miguel-ch** [reply]
How to achieve this behavior?

**Reply by urielconvey** [reply]
Hi @mikkhait, how did you achive that? i've been trying to do it for hours.
Thanks!
